README.md

```diff
@@ -1,8 +1,19 @@
 [![Build and Test](https://github.com/dafny-lang/dafny/workflows/Build%20and%20Test/badge.svg)](https://github.com/dafny-lang/dafny/actions?query=workflow%3A%22Build+and+Test%22) [![Gitter](https://badges.gitter.im/dafny-lang/community.svg)](https://gitter.im/dafny-lang/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
+Dafny is a **programming language with a program verifier**. As you type in your program, the verifier constantly looks over your shoulders, flags any errors, shows you counterexamples, and congratulates you when your code matches your specifications. When you're done, Dafny **compiles your code to C#, Java, JavaScript and Go**, so it can integrate in your existing workflow.
+
+![vs-code-dafny-2 0 0-demo](https://user-images.githubusercontent.com/3601079/140799975-f3ac0925-10d9-4c14-b1a9-cd449854c6ae.gif)
+
+Developers use Dafny to obtain **guarantees that their code meets the specifications they write**, by easily writing both code and specifications in the Dafny programming language itself, and then **spending far less time hinting Dafny to verify the code** than chasing corner bugs down the road.
```

Comment:
Since it's not just a vague "using Dafny", it's really related to writing specifications so that the verifier can find a proof, what about the following alternative?
**spending far less time elaborating specifications to verify the code**

I think the change of tone is appropriate. The first paragraph is giving everyday context on how it would apply to the reader who is assumed to be a new user of Dafny (who else will look at the README? :-) ), and the second paragraph transitions into the experience other developers had in using Dafny, before switching to the third paragraph where we don't talk about developers at all, only about even more general mathematical concepts.
On a website, I would argue to use "you" everywhere, but for this README I'm fine doing this gradual transition.